mitiq/mitiq_qiskit/tests/test_zne_mitiq_qiskit.py

```diff
@@ -49,103 +41,113 @@ def measure(circuit, qid) -> QuantumCircuit:
     """
     # Ensure that we have a classical register of enough size available
     if len(circuit.clbits) == 0:
-        reg = ClassicalRegister(qid + 1, "creg")
+        reg = ClassicalRegister(qid + 1, "cbits")
```

Comment:
Note `creg` is a keyword in Qasm. This was causing bugs in the conversions.

Reply:
nice catch!